testem.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = {
-  test_page: 'tests/index.html?hidepassed',
+  test_page: 'tests/index.html?hidepassed&nolint',
```

Comment:
We shouldn't need to add the `nolint` query param because we're no longer using `ember-cli-qunit` or `ember-cli-template-lint` in this project. The test server should only be running the ember tests.